input.yml

```diff
@@ -20,6 +20,7 @@
       - { name: Google Compute Engine, alias: gce }
       - { name: Scaleway, alias: scaleway}
       - { name: OpenStack (DreamCompute optimised), alias: openstack }
+      - { name: CloudStack, alias: cloudstack }
```

Comment:
Would it be necessary to tell the user that the module is Exoscale optimised? I'm not quite familiar with CloudStack, but I think configs may vary among the providers (like in OpenStack)

Reply:
indeed, they seem to be the ones maintaining the ansible module :)